flixel/input/FlxKeyManager.hx

```diff
@@ -5,6 +5,11 @@ import flixel.FlxG;
 import flixel.input.FlxInput.FlxInputState;
 import flixel.input.keyboard.FlxKey;
 
+/**
+ * Keeps track of what keys are pressed and how with handy Bools or strings.
+ * Automatically instatiated by flixel via `FlxKeyboard` and accessed via `FlxG.keys`.
```

Comment:
Not only that, also android keys.

Reply:
is that automatically instantiated on android, how is it accessed?

Reply:
FlxG.android :D